lib/plugins/features/movies/controller.js

```diff
@@ -27,3 +30,19 @@ exports.list = (query) => {
   })
   .fetchAll();
 };
+
+exports.addLocation = (payload) => {
+  return Bluebird.all([
+    new Location({ id: payload.location_id }).fetch(),
```

Comment:
Good question. We definitely need to do a better job of explaining the "why".

So throughout our API, there are several payload params like this (that accept an id). However, there are also some of them that accept both an id or something else. This might be an object to be able to create it inline (e.g. address id or inline address), or it might be a URL to an asset (e.g. template id or HTML string or remote URL).

As opposed to specifying these params explicitly as `_id`s, we prefer to think of them as the resource. You're not passing in a "to address" _id_ you're passing in some reference to a "to address".

Does that make sense?